src/constraints/create.jl

```diff
@@ -0,0 +1,174 @@
+export compute_constraints_indices
+
+function compute_constraints_indices(connection)
+    constraints = Dict{Symbol,TulipaConstraint}()
+
+    constraints[:lowest] = TulipaConstraint(
```

Comment:
This still uses the old table names. Miro shows the tables after renaming and some splitting, which hasn't happened yet to avoid further complications. There are some purple stickers in the bottom of the constraint tables showing where they come from.